src/site/content/en/blog/ecommerce-notifications-best-practice/index.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Use best practices for e-commerce notifications
+subhead: Push notifications can help you engage with your customers. Here's how to do them right. 
+authors:
+  - katejeffreys
+date: 2019-07-30
+hero: hero.jpg
+alt: A man looking at his phone at sunset. Photo by @brunoreyna on unsplash.com.
+description: |
+  Learn best practices for implementing push notifications in an e-commerce web app.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - notifications
+  - push
+  - e-commerce
+---
+
+When used effectively, push notifications help your customers to get the most
+out of their online shopping experience. 
+
+Targeted, timely, respectful notifications help people to engage more deeply
+with your site, and build trust in your brand–leading to satisfied customers
+and more repeat shoppers.
+
+With push notifications:
+
+*   Remind a customer about something they need to do–for example, finalize
```

Comment:
Nit: for each of these, I'd be inclined to start a new sentence rather than use a dash: '... something they need to do. For example: ...'